### PR TITLE
Add a new 'Styling' category to the 'Documentation issue' template

### PR DESCRIPTION
**Pull request by aSemy**
Adds a new 'Styling' category to the 'Documentation issue' template.

### Context

Otherwise it's not clear what issue should be selected. The other categories don't seem related.

For example, these issues:

* #27617
* #27623 

### Contributor Checklist
- [ ] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [ ] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [ ] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [ ] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [ ] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [ ] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [ ] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [ ] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [ ] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/40_contributor_documentation.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ body:
       options:
         - Wrong or misleading information
         - Missing information
+        - Styling
```
[the remark this answers is not in the document]
Comment (ov7a) [reply .github/ISSUE_TEMPLATE/40_contributor_documentation.yml:21]: Yeah, I think it's better as 'Styling or Accessibility'

**Review by ov7a — APPROVED** [review]

**Comment by ov7a** [comment]
Thank you for your contribution!

The [DCO](https://probot.github.io/apps/dco/) check is failing, please look into that as we can't accept the PR otherwise.

**Comment by aSemy** [comment]
Sorry, I don't have an option to add a DCO.

**Comment by ov7a** [comment]
> Sorry, I don't have an option to add a DCO.

I assume you created this PR via Github UI and we didn't have an automatic DCO. It's fixed now.

Because of legal reasons, I can't merge this PR (😵‍💫). Are you ok with me just copying it?  Or you can close this PR and redo the change, it should be signed off.

**Comment by ov7a** [comment]
Closing in favor of #27691

**Pull request closed without merging**